### PR TITLE
Reorder option type edit

### PR DESCRIPTION
**Pull request by Murph33**
Currently the option type edit page is confusing as there is one legend and it is for option values while it includes the option type attributes.  This will fix that.

Before (confusing)
before
After (better)
after

@cbrunsdon Makes total sense will make sure to attach these in future visual change PRs

### CODE REVIEW

**Comment by Senjai** [comment]
Good small commits :+1: (best reviewed with `?w=1`)

**Comment by Murph33** [comment]
I've realized the actual placements could be a bit better going to rebase them in

**Comment by cbrunsdon** [comment]
Looks reasonable to me, however images are always appreciated of the before/after too. :+1:

File: backend/app/views/spree/admin/option_types/edit.html.erb [hunk 1]
```diff
@@ -16,11 +16,12 @@
 <%= render :partial => 'spree/shared/error_messages', :locals => { :target => @option_type } %>
 
 <%= form_for [:admin, @option_type] do |f| %>
-  <fieldset>
-    <legend align="center"><%= Spree.t(:option_values) %></legend>
-
+  <fieldset class="no-border-bottom">
+    <legend align="center"><%= Spree::OptionType.model_name.human %></legend>
```
Comment (mamhoff) [inline backend/app/views/spree/admin/option_types/edit.html.erb]: I don't think we need the `align="center"` here. CSS should do that for us.

File: backend/app/views/spree/admin/option_types/edit.html.erb [hunk 2]
```diff
@@ -16,11 +16,12 @@
...
     <%= render :partial => 'form', :locals => { :f => f } %>
-
+  </fieldset>
+  <fieldset class="no-border-bottom">
+    <legend align="center"><%= Spree.t(:option_values) %></legend>
```
Comment (mamhoff) [inline backend/app/views/spree/admin/option_types/edit.html.erb]: Same, no presentation in markup.

**Comment by mamhoff** [comment]
Can you rebase on master? I've also added a small nitpick that's easily cured.

File: backend/app/views/spree/admin/option_types/edit.html.erb [hunk 3]
```diff
@@ -16,11 +16,12 @@
...
 <%= form_for [:admin, @option_type] do |f| %>
-  <fieldset>
-    <legend align="center"><%= Spree::OptionValue.model_name.human(count: :other) %></legend>
-
+  <fieldset class="no-border-bottom">
+    <legend><%= Spree::OptionType.model_name.human %></legend>
     <%= render :partial => 'form', :locals => { :f => f } %>
-
+  </fieldset>
+  <fieldset class="no-border-bottom">
```
Comment (mamhoff) [inline backend/app/views/spree/admin/option_types/edit.html.erb]: Sorry, I've been playing with these fieldsets all day long - and I think this one should actually have a bottom border as it's the last fieldset on the page. Would you mind amending this again?

**Comment by mamhoff** [comment]
Thanks!

**Pull request merged**